src/SynchronousMessageDispatcher.php

```diff
@@ -7,11 +7,11 @@
 final class SynchronousMessageDispatcher implements MessageDispatcher
 {
     /**
-     * @var Consumer[]
+     * @var MessageConsumer[]
```

Comment:
Probably better to declare this as `list<MessageConsumer>`, if phpstan understands it